prov/verbs/src/verbs_info.c

```diff
@@ -203,9 +203,11 @@ int fi_ibv_check_rx_attr(const struct fi_rx_attr *attr,
 	uint64_t saved_prov_mode = info->rx_attr->mode;
 	int ret;
 
-	info->rx_attr->mode = hints->domain_attr->cq_data_size ?
-		info->rx_attr->mode :
-		(info->rx_attr->mode & ~FI_RX_CQ_DATA);
+	if (hints->domain_attr) {
+		info->rx_attr->mode = hints->domain_attr->cq_data_size ?
+			info->rx_attr->mode :
+			(info->rx_attr->mode & ~FI_RX_CQ_DATA);
+	}
```

Comment:
Thanks. We would have to zero FI_RX_CQ_DATA bit in case hints->domain_attr is NULL as well. So having an additional check for hints->domain_attr in the ternary operator should do it.

Reply:
Ok. Thanks for the comment